sdk/core/logger/src/debug.ts

```diff
@@ -153,21 +166,11 @@ function destroy(this: Debugger): boolean {
 }
 
 function extend(this: Debugger, namespace: string): Debugger {
+  // eslint-disable-next-line no-invalid-this
```

Comment:
It's a little disappointing that eslint complains here, given we are typing `this` above, but I suppose this is a defect of ESLint not fully understanding TS.

Since this occurs 3 times in this file and repeated ignores add little value, perhaps we should just disable the rule for the entire file?